src/app/story/directives/hero.component.ts

```diff
@@ -46,6 +47,12 @@ import { StoryModel } from '../../shared';
              [working]="story.isPublishing" (click)="togglePublish()" orange=1>
             {{story.publishedAt ? 'Unpublish' : 'Publish'}}
           </publish-button>
+          <p *ngIf="!story.changed() && !story.publishedAt">
+            Will be queued up for publishing {{story.releasedAt ? ' at ' + formatDate(story.releasedAt) : 'immediately'}}
```

Comment:
I don't love this language. It's kind of long and maybe not the best. Any better suggestions? 
 - Does it need to say anything at all when the release date is not set? Maybe it should just say so when the release date is set since that is the anomaly. Otherwise, seems obvious that Publish means now without saying so.
 - To shorten it a bit, maybe show date without time? Time may not be necessary here since it is on the page with the releasedAt date